src/app/subject/services/volume-data.service.ts

```diff
@@ -0,0 +1,74 @@
+import { HttpClient, HttpHeaders } from '@angular/common/http'
+import { Injectable } from '@angular/core'
+import { Actions, ofType } from '@ngrx/effects'
+import { Action } from '@ngrx/store'
+import { Observable } from 'rxjs'
+import { takeUntil } from 'rxjs/operators'
+
+import { ENV } from '../../../environments/environment'
+import { DescriptiveStatistic } from '../../shared/enums/descriptive-statistic.enum'
+import { Source } from '../../shared/models/source.model'
+import { Subject } from '../../shared/models/subject.model'
+import * as volumeDataActions from '../store/volume-data/volume-data.actions'
+
+@Injectable()
+export class VolumeDataService {
+  private url = `${ENV.API_URI}/aggregate`
+  private destroy$: Observable<Action>
+
+  // NOTE: Temporary values are used to get data from volume API. Time window is automatically specified (ONE_WEEK) by API.
+  private endTime = '2018-04-27T14:00:00Z'
```

Comment:
This value that can be computed by API too, if not specified. 
Despite, can we calculate the time based on `new Date()` and corresponding startTime rather hard-coded values

Reply:
Okay sure.